.github/ISSUE_TEMPLATE/---bug.md

```diff
@@ -0,0 +1,44 @@
+---
+name: "\U0001F41B Bug"
+about: Create a report to help us improve
+title: ''
+labels: "\U0001F41B bug, ux & visual design"
```

Comment:
```suggestion
labels: "\U0001F41B bug, ux-and-visual-design"
```

I have updated the label to remove spaces to avoid issues with the add issue to project GH Action. THis suggestion aligns to that.